O/OpenMPI/build_tarballs.jl

```diff
@@ -77,9 +85,20 @@ products = [
 
 dependencies = [
     Dependency("CompilerSupportLibraries_jll"),
+    Dependency("Hwloc_jll"),    # compat="2.0.0"
+    Dependency("PMIx_jll"),     # compat="4.2.0"
+    Dependency("Zlib_jll"),
+    Dependency("libevent_jll"), # compat="2.0.21"
+    Dependency("prrte_jll"),    # compat="3.0.0"
```

Comment:
#7606 I am not sure that using the pprte_jll and PMIx_jll were ready to be fully used as dependencies. I recall having to do weird things to make them work.

https://github.com/JuliaParallel/PMIx.jl/blob/b5c6be4fe342bae7fc6ee770e1e5cc9d388ceab1/src/api.jl#L11